src/command.c

```diff
@@ -403,7 +404,7 @@ static int dv_convert_conv_v0(struct device *dev, struct dmp_cmb *cmb,
 		return -EINVAL;
 	}
 	/*pr_info(DRM_DEV_NAME ": tiles=%d for %dx%dx%d\n",
-		(int)conv->input.tiles, (int)conv->input.w, (int)conv->input.h, (int)conv->input.c);*/
+	  (int)conv->input.tiles, (int)conv->input.w, (int)conv->input.h, (int)conv->input.c);*/
```

Comment:
was correct before as it is in comment and when comment is removed - it will became properly aligned
another solution => start comment on the line before